### ISSUE TITLE
Bring back the time markers

### ISSUE THREAD

**Issue by duckythescientist** [miek/inspectrum#67]
Older builds had the option to enable markers on the time axis. The newest build of master seems to be lacking this feature. I used those time markers with a separate script to trim sections of a file. Even after #49 gets implemented, it would be nice to have those markers back.

**Comment by miek** [comment]
It's back in master thanks to @schneider42 & #79
